app/helpers/grades_helper.rb

```diff
@@ -36,14 +33,68 @@ def has_team_and_metareview?
     {has_team: has_team, has_metareview: has_metareview, true_num: true_num}
   end
 
+  def participant
+    # returns a participant given an id
+    @participant = Participant.find(params[:id])
+    @participant
+  end
+
+  def pscore
+    # returns participant scores
+    @participant = Participant.find(params[:id])
+    @pscore = @participant.scores(@questions)
+    @pscore
+  end
+
+  def rscore_review
```

Comment:
For this method and next three methods, I think it will be cleaner to use metaprogramming in ruby with keywords `eval`.